src/server/JasmineGraphInstance.cpp

```diff
@@ -50,11 +50,122 @@ int JasmineGraphInstance::start_running(string profile, string hostName, string
     pthread_create(&instanceCommunicatorThread, NULL, runInstanceService, this);
     pthread_create(&instanceFileTransferThread, NULL, runFileTransferService, this);
 
+    acknowledgeMaster(masterHost,hostName,std::to_string(serverPort));
+
     pthread_join(instanceCommunicatorThread,NULL);
     pthread_join(instanceFileTransferThread,NULL);
 
     }
 
+bool JasmineGraphInstance::acknowledgeMaster(string masterHost, string workerIP, string workerPort) {
+    int sockfd;
+    char data[300];
+    bool loop = false;
+    socklen_t len;
+    struct sockaddr_in serv_addr;
+    struct hostent *server;
+    Utils utils;
+
+    sockfd = socket(AF_INET, SOCK_STREAM, 0);
+
+    if (sockfd < 0) {
+        std::cerr << "Cannot accept connection" << std::endl;
+        return 0;
+    }
+
+    if (masterHost.find('@') != std::string::npos) {
+        masterHost = utils.split(masterHost, '@')[0];
+    }
+
+    graphInstance_logger.log("###FRONTEND### Get Host By Name : " + masterHost, "info");
```

Comment:
In the log message it is mentioned `FRONTEND`. However, this is JasmineGraphInstance (i.e., worker). Hence lets use `INSTANCE` instead of `FRONTEND`.

Reply:
I'll change the log message accordingly

Reply:
Ok